src/commands/lambda/constants.ts

```diff
@@ -135,18 +135,30 @@ export const SKIP_MASKING_ENV_VARS = new Set([
   VERSION_ENV_VAR,
 ])
 
+export enum Frameworks {
```

Comment:
```suggestion
export enum DeploymentFrameworks {
```

I think `Framework` is not very specific